docs/course_materials/exercises/test_exercises/03_Variables_and_datatypes/test_homework_variables.py

```diff
@@ -0,0 +1,73 @@
+"""
+Automatically check student's homework
+----------------------------------------
+The tests will check:
+ 1. The content of the exercise file
+ 2. The execution and output of the exercise file
+ 3. The output with different inputs
+
+Guidelines:
+- You have to specify the dir Student's exercise files should be placed
+in the same dir as this test.
+- Test names are in the form: test_<exercise>.py
+- Before each test should be the appropriate fixture: @pytest.mark.<topic>
+"""
+import pytest
+
+
+@pytest.mark.variables
+def test_variables_1(helpers):
+    helpers.student_file = helpers.home_dir + 'variables_1.py'
+    helpers.expected_pycode = [
+        [r'^.*x *= *9.*', 'x variable is not defined'],
```

Comment:
I cheked this code (which one of the students submitted and it is good) : 
x = int(9)
y = int(7)
z = x + y 
print(z)

and I got the warnings: 
x variable is not defined
y variable is not defined

Another code is: x,y=int(9),int(7) which also correct and get the same error.
same for z if it defined: z=int(x+y)
(I agree it's better to write without the int(), but maybe we can write a better message for this) 

also if I will use x = 19 I will still get x variable is not defined
which I believe is not a good message, because x is defined but not with 9.